arithmetization/src/test/java/net/consensys/linea/zktracer/module/mxp/MxpTest.java

```diff
@@ -296,6 +304,218 @@ void testCall(TestInfo testInfo) {
     toyExecutionEnvironmentV2.run();
   }
 
+  @Tag("nightly")
+  @ParameterizedTest
+  @MethodSource({"testMxpxThresholdSource"})
+  void testMxpxThreshold(
+      OpCode opCode,
+      BigInteger offset1,
+      BigInteger offset2,
+      BigInteger size1,
+      BigInteger size2,
+      TestInfo testInfo) {
+    BytecodeCompiler program = BytecodeCompiler.newProgram(chainConfig);
+
+    switch (opCode) {
+        // 1 offset
+      case MSTORE, MSTORE8 -> program
+          .push(0) // value
+          .push(offset1)
+          .op(opCode);
+        // 2 offsets, 1 size
+      case CODECOPY -> program.push(size1).push(offset2).push(offset1).op(opCode);
+        // 2 offsets, 2 sizes
+      case CALL -> program
+          .push(size2)
+          .push(offset2)
+          .push(size1)
+          .push(offset1)
+          .push(0) // value
+          .push(0) // address
+          .push(Bytes.fromHexStringLenient("0xFFFFFFFF")) // gas
+          .op(opCode);
+      default -> throw new IllegalArgumentException("Unsupported opCode: " + opCode);
+    }
+
+    BytecodeRunner.of(program.compile()).run(chainConfig, testInfo);
+  }
+
+  static final BigInteger MAX_UINT256 =
+      BigInteger.TWO.pow(256).subtract(BigInteger.ONE); // 2^256 - 1
+  static final BigInteger LONDON_MXPX_THRESHOLD = (BigInteger.valueOf(256).pow(4));
+  static final BigInteger CANCUN_MXPX_THRESHOLD =
+      (BigInteger.valueOf(256).pow(4)).subtract(BigInteger.ONE);
+  static final BigInteger SMALL = BigInteger.valueOf(32);
+
+  static final List<OpCode> oneOffsetOpCodes = List.of(OpCode.MSTORE, OpCode.MSTORE8);
+  static final List<OpCode> twoOffsetsOneSizeOpCodes = List.of(OpCode.CODECOPY);
```

Comment:
Same issue: only `MCOPY` is a 2 offsets, 1 size opcode from the MXP point of view.